src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -128,7 +128,7 @@ export class SessionSubmissionPageComponent implements OnInit {
       this.courseId = queryParams.courseid;
       this.feedbackSessionName = queryParams.fsname;
       this.regKey = queryParams.key ? queryParams.key : '';
-      this.moderatedPerson = queryParams.moderatedperson ? queryParams.moderatedperson : '';
+      // this.moderatedPerson = queryParams.moderatedperson ? queryParams.moderatedperson : '';
```

Comment:
Will this cause any side-effect? If you delete this, you can also delete L130?

Reply:
Sry! I realised I wrongly commented this out. Its supposed to be as it is. Both L131/L130 should be there.